src/components/Contentful/Hero.js

```diff
@@ -33,16 +42,17 @@ export default function ContentfulHero({
   return (
     <div className="contentful-hero" id={id}>
       <div className="container">
-        <div className="row">
-          <div className="col-xl-12 col-lg-12 col-md-12 col-sm-12 col-12">
+        <div className="row contentful-hero__row">
```

Comment:
I recommend using nested or wrapper divs rather than combining bootstrap classes and custom classes in the same element. You don't want to be meddling with selector specificity issues later on.